pom.xml

```
@@ -27,6 +28,37 @@
             <artifactId>spring-boot-starter-test</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.h2database</groupId>
+            <artifactId>h2</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
```

I think if you have a hibernate as a separate dependency - a simple `spring-boot-starter` should be enough so far
```suggestion
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter</artifactId>
```